onnxruntime/python/tools/kernel_explorer/kernels/utils.py

```diff
@@ -100,3 +101,28 @@ def softmax(x, *, is_log_softmax=False, axis=-1):
     if is_log_softmax:
         return x - np.log(np.sum(np.exp(x), axis=axis, keepdims=1))
     return (np.exp(x)) / np.sum(np.exp(x), axis=axis, keepdims=1)
+
+
+def _matmul(a, b):
+    if os.getenv("KERNEL_EXPLORER_TEST_USE_CUPY", "0") == "1":
+        import cupy as cp
+
+        return (cp.asarray(a) @ cp.asarray(b)).get()
+    else:
+        return a @ b
+
+
+def matmul(a, b, transa=False, transb=False):
+    return _matmul(a.T if transa else a, b.T if transb else b)
+
+
+def fast_gelu(x, bias):
+    x = x + bias
+    y = 0.5 * x * (1 + np.tanh(0.797885 * x + 0.035677 * x * x * x))
+    return y
+
+
+def normalization(x, axis, epsilon):
```

Comment:
This is not normalization, it is standardization.

Reply:
Done.